DesignSafe-CI/portal
# Bug/DES-2927: Hyb Sim showing wrong View Overview modal

## Overview: ##
For project type Hybrid Simulation, the 'View Overview' button was opening to the Best Practices modal. 
Apparently Hybrid Simulation doesn't have a 'View Overview' modal. I added a page that reflects what is in PROD, which says "Hybrid Simulation Overview Coming Soon...".
## PR Status: ##


## Related Jira tickets: ##

* [DES-2927](https://tacc-main.atlassian.net/browse/DES-2927)

## Summary of Changes: ##
Added a HybridSimulationSteps.tsx page
Linked it to ProjectTypeModal.tsx

## Testing Steps: ##
1. In My Projects, open a Hybrid Simulation project. Click 'View Overview' Verify the modal is the Overview modal, not Best Practices. 

## UI Photos:
<img width="812" alt="Screen Shot 2024-06-26 at 9 23 31 AM" src="https://github.com/DesignSafe-CI/portal/assets/35277477/edc457f2-c3bb-42f1-950f-e48b27cdf624">

## Notes: ##
